fluid/resnet50.py

```diff
@@ -170,12 +170,18 @@ def run_benchmark(model, args):
 
     iter = 0
     im_num = 0
+    start_time = time.time()
     for pass_id in range(args.pass_num):
+        every_pass_loss = []
+        every_pass_acc = []
         accuracy.reset(exe)
         if iter == args.iterations:
-            break
+            iter = 0
```

Comment:
Is this modification right? What's the use of `args.iterations` ?

Reply:
I think it shoud be:
accuracy.reset(exe)
iter = 0
the args.iterations is used to  limit the iterations in each pass, so it used in:
for batch_id, data in enumerate(train_reader()):